### PR TITLE
Feature/assign tasks to users [UI]

### PR DESCRIPTION
**Pull request by aindreias**
Users can now assign tasks to other people in their household.

Currently this is not backed up on firebase, so the household member list is static.

I plan on:
- merging this with my modifications from the test branch (✓), then
- simplifying / cleaning the remaining code (✓) (#139)

### CODE REVIEW

File: app/src/main/java/com/github/houseorganizer/houseorganizer/task/TaskListAdapter.java [hunk 1]
```diff
@@ -81,6 +96,7 @@ public void onBindViewHolder(@NonNull BiViewHolder<Button, Button> holder, int p
             final AlertDialog alertDialog
                     = new AlertDialog.Builder(v.getContext())
                     .setNeutralButton(R.string.add_subtask, null)
+                    .setPositiveButton("Assignees", null) // TODO extract string
```
Comment (Arkadyus) [inline app/src/main/java/com/github/houseorganizer/houseorganizer/task/TaskListAdapter.java:99]: that TODO should probably be done before merging

**Review by Arkadyus — CHANGES_REQUESTED** [review]
Looks really clean, just a todo to implement.
Im assuming testing will come in another sprint ? We should probably starting putting testing in each of our tasks by default soon though

File: app/src/main/java/com/github/houseorganizer/houseorganizer/task/TaskListAdapter.java (hunk 1 above)
Comment (aindreias) [reply app/src/main/java/com/github/houseorganizer/houseorganizer/task/TaskListAdapter.java:99]: Thanks for catching it! Fixing it in the next commit

**Comment by aindreias** [comment]
> Looks really clean, just a todo to implement. Im assuming testing will come in another sprint ? We should probably starting putting testing in each of our tasks by default soon though

I might still have time to test the RecyclerViews tonight. I expect UI testing on my part to be done by next sprint

**Review by Arkadyus — APPROVED** [review]
Perfect, hopefully we can make this work in tandem with firebase soon

**Review by Geno22 — APPROVED** [review]
Nothing to say amazing work !
Must have been a pain to rename everything to HTask xD

**Comment by aindreias** [comment]
> Nothing to say amazing work ! Must have been a pain to rename everything to HTask xD

That was, to be fair, easier than I expected

**Pull request merged**